doc/porting_guide.md

```diff
@@ -1,5 +1,6 @@
 # Google Cloud IoT Device SDK for Embedded C Porting Guide
-##### Copyright 2018-2019 Google LLC
+
```

Comment:
nit: Looks like a space snuck in here

Reply:
the new line?

Reply:
was added intentionally, to be similar to its company a file user_guide.md, next to it